src/gameboy/main.cpp

```diff
@@ -85,8 +85,20 @@ int main(int argc, char *argv[]) {
 	}
 
 	auto cartridge = std::make_unique<Cartridge>(rom_path);
+
+	/**
```

Comment:
Why are you doing this here if this is being done already in the cartridge module?